From: Jelmer Vernoo캐 <jelmer@debian.org>
Subject: Automatically generated patch (0.0.6-2)

Last (up to) 3 git changes, FYI:

commit 0a558438084f443c0088af5a93743ea085477374
Author: Jelmer Vernoo캐 <jelmer@jelmer.uk>
Date:   Tue Jul 18 00:59:20 2017 +0000

    releasing package xandikos version 0.0.6-2

commit 2e97990520cfbf67b2993d337ed642ca6d09a890
Author: Jelmer Vernoo캐 <jelmer@jelmer.uk>
Date:   Tue Jul 18 00:59:06 2017 +0000

    When running autopkgtest tests, use system xandikos binary.

commit 209cca45ce66e511ff2cd11e99bed8a3fe125489
Author: Jelmer Vernoo캐 <jelmer@jelmer.uk>
Date:   Thu Jul 13 23:03:40 2017 +0000

    releasing package xandikos version 0.0.6-1
---

--- xandikos-0.0.6.orig/.travis.yml
+++ xandikos-0.0.6/.travis.yml
@@ -1,29 +1,12 @@
-language: python
-cache: pip
-sudo: true
-python:
-  - 3.3
-  - 3.4
-  - 3.5
-  - 3.6
-  - pypy3.3-5.2-alpha1
-env:
-  global: PYTHONHASHSEED=random
-install:
-  - pip install pip --upgrade
-  - pip install coverage codecov flake8 pycalendar
-  - sudo apt-get install -qq libneon27-dev curl python2.7
-  - python setup.py develop
+sudo: required
+language: generic
+
+services:
+  - docker
+
 script:
-  - make style
-  - make coverage
-  - mv .coverage .coverage.unit
-  - make coverage-litmus
-  - mv .coverage .coverage.litmus
-  - make coverage-vdirsyncer
-  - mv .coverage .coverage.vdirsyncer
-  - make coverage-caldavtester
-  - mv .coverage .coverage.caldavtester
-after_success:
-  - python -m coverage combine
-  - codecov
+  - wget -O- http://travis.debian.net/script.sh | sh -
+
+branches:
+  except:
+    - /^debian\/\d/
